handler/src/main/java/io/netty5/handler/stream/ChunkedInput.java

```diff
@@ -31,6 +30,7 @@
     /**
      * Releases the resources associated with the input.
      */
+    @Override
```

Comment:
why we need this override at all ?

Reply:
The override is only there for that one line of javadoc.

Reply:
Removing it.